research/format.markdown

```diff
@@ -0,0 +1,124 @@
+| Added       | `uuid`             | A string instance is valid against this attribute if it is a valid string representation of a UUID, according to [RFC4122] |
+
+### Changes :
+
+| Keyword   | Previous Description | Present Description |
+|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| `hostname`  | As defined by RFC 1034, section 3.1 [RFC1034], including host names produced using the Punycode algorithm specified in RFC 5891, section 4.4 [RFC5891]. | As defined by RFC 1123, section 2.1, including host names produced using the Punycode algorithm specified in RFC 5891, section 4.4 [RFC5891] |
+
+## 2019-09 to 2020-12
+
+References: [2019-09](https://json-schema.org/draft/2019-09/draft-handrews-json-schema-validation-02#rfc.section.7), [2020-12](https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-01#section-7)
+
+> Behaviour: 
+      >>[Before](https://json-schema.org/draft/2019-09/draft-handrews-json-schema-validation-02#rfc.section.7.2.3): An implementation MUST NOT fail validation or cease processing due to an unknown format attribute.  
+      [After](https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-01#section-7.2.3): When the Format-Assertion vocabulary is specified, implementations MUST fail upon encountering unknown formats.
+
+### Changes :
+
+| Keyword    | Previous Description                                                                                                    | After Description                                                                                                           |
+|------------|--------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| `email`      | Defined in RFC 5321                                                                                                      | As defined by the `Mailbox` ABNF rule in RFC 5321, section 4.1.2 [RFC5321]                                                |
+| `idn-email`  | RFC 6531                                                                                                                 | As defined by the extended `Mailbox` ABNF rule in RFC 6531, section 3.3 [RFC6531]                                         |
+| `regex`      | ECMA-262                                                                                                                 | ECMA-262 defined in the Regular Expressions (Section 4.3)                                                                |
```

Comment:
Isn't this the same? Last time you mentioned `regex` it was still an ECMA-262 one?

Reply:
Yeah, just that section was specified explicitely. So these changes arent necessary? to be mentioned

Reply:
Correct. Let's avoid these changes that are only RFC clarifications without semantic impact to the keywords

Reply:
Okay..  then there won't be any changes as all changes I mentioned were RFC changes